### PR TITLE
Some changes that were applied straight to PROD now in DEV

### PR DESCRIPTION
**Pull request by carioca-au**

### CODE REVIEW

File: dags/landsat_scenes_sync/landsat_scenes_gap_report.py [hunk 1]
```diff
@@ -288,10 +282,10 @@ def generate_buckets_diff(landsat: str, file_name: str, update_stac: bool = Fals
         )
 
         logging.info(f"Number of missing scenes: {len(missing_scenes)}")
-        logging.info(f"Wrote missing scenes to: s3://{LANDSAT_SYNC_BUCKET_NAME}/{key}")
+        logging.info(f"Wrote missing scenes to: {LANDSAT_SYNC_BUCKET_NAME}/{key}")
 
         if len(orphaned_scenes) > 0:
-            output_filename = f"{landsat}_{date_string}_orphaned.txt.gz"
+            output_filename = f"{landsat}_{datetime.today().isoformat()}_orphaned.txt.gz"
```
Comment (alexgleith) [inline dags/landsat_scenes_sync/landsat_scenes_gap_report.py:288]: Can we please change this back to using a simple date.

File: dags/landsat_scenes_sync/landsat_scenes_gap_report.py [hunk 2]
```diff
@@ -288,10 +282,10 @@ def generate_buckets_diff(landsat: str, file_name: str, update_stac: bool = Fals
         )
 
         logging.info(f"Number of missing scenes: {len(missing_scenes)}")
-        logging.info(f"Wrote missing scenes to: s3://{LANDSAT_SYNC_BUCKET_NAME}/{key}")
+        logging.info(f"Wrote missing scenes to: {LANDSAT_SYNC_BUCKET_NAME}/{key}")
```
Comment (alexgleith) [inline dags/landsat_scenes_sync/landsat_scenes_gap_report.py:285]: Please include the S3 URI prefix

File: dags/landsat_scenes_sync/landsat_scenes_gap_report.py [hunk 3]
```diff
@@ -301,7 +295,7 @@ def generate_buckets_diff(landsat: str, file_name: str, update_stac: bool = Fals
                 content_type="application/gzip"
             )
             logging.info(f"Number of orphaned scenes: {len(orphaned_scenes)}")
-            logging.info(f"Wrote orphaned scenes to: s3://{LANDSAT_SYNC_BUCKET_NAME}/{key}")
```
Comment (alexgleith) [inline dags/landsat_scenes_sync/landsat_scenes_gap_report.py:304]: Please include the S3 URI prefix

File: dags/utils/inventory.py [hunk 4]
```diff
@@ -66,7 +66,7 @@ def latest_manifest(self, key: str = "", suffix: str = ""):
         Return a dictionary of a manifest file"
         """
         logging.info(
-            f"Start Looking for latest manifest file in the bucket: {self.bucket_name}"
```
Comment (alexgleith) [inline dags/utils/inventory.py:69]: Please change this back to the new phrasing

File: dags/utils/inventory.py [hunk 5]
```diff
@@ -78,7 +78,9 @@ def latest_manifest(self, key: str = "", suffix: str = ""):
             manifest_paths = [k for k in self.find(suffix=suffix, sub_key=_key)]
 
             if len(manifest_paths) == 1:
-                logging.info(f"Latest Manifest {manifest_paths[0]}")
```
Comment (alexgleith) [inline dags/utils/inventory.py:81]: Please change this back

File: dags/utils/inventory.py [hunk 6]
```diff
@@ -102,7 +104,7 @@ def retrieve_manifest_files(self, key: str = "", suffix: str = ""):
         manifest = self.latest_manifest(key=key, suffix=suffix)
 
         logging.info(
-            f"Retrieved Manifest with {len(manifest['files'])} Files"
```
Comment (alexgleith) [inline dags/utils/inventory.py:105]: Please change this back

**Review by alexgleith — CHANGES_REQUESTED** [review]
It's not passing linting and I have a range of changes that I think we should not be making.

File: dags/landsat_scenes_sync/landsat_scenes_gap_report.py (hunk 1 above)
Comment (carioca-au) [reply dags/landsat_scenes_sync/landsat_scenes_gap_report.py:288]: I thought you had added that... sorry

**Pull request closed without merging**